src/uct/tcp/tcp_cm.c

```diff
@@ -118,7 +102,7 @@ static ucs_status_t uct_tcp_cm_send_conn_req(uct_tcp_ep_t *ep)
         uct_tcp_cm_trace_conn_pkt(ep, "unable to send connection request to",
                                   &ep->peer_addr);
         if (status == UCS_ERR_IO_ERROR) {
-            uct_tcp_cm_log_somaxconn_exceed();
+            ucs_error("try to increase \"net.core.somaxconn\" on the remote node");
```

Comment:
can we avoid code duplication with uct_tcp_cm_recv_conn_ack()?

Reply:
Is it OK to implement a wrapper for this `ucs_error(...)` and use in the both places?

Reply:
@yosefe probably the last commit is what we want